sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LegacyServiceClient.cs

```diff
@@ -16,87 +19,1378 @@ namespace Azure.AI.TextAnalytics.ServiceClients
     ///
     internal class LegacyServiceClient : ServiceClient
     {
+        private static readonly TextAnalyticsRequestOptions DefaultRequestOptions = new();
+        private static readonly RecognizePiiEntitiesOptions DefaultPiiEntitiesOptions = new();
+        private static readonly AnalyzeHealthcareEntitiesOptions DefaultHeathcareEntitiesOptions = new();
+        private static readonly AnalyzeSentimentOptions DefaultAnalyzeSentimentOptions = new();
+
+        internal readonly TextAnalyticsRestClient _serviceRestClient;
+        internal readonly ClientDiagnostics _clientDiagnostics;
+
+        private readonly Uri _baseUri;
+        private readonly string _defaultLanguage;
+        private readonly string _defaultCountryHint;
+
+        public LegacyServiceClient(Uri endpoint, TokenCredential credential, string defaultLanguage, string defaultCountryHint, string authorizationScope, string serviceVersion, ClientOptions options)
```

Comment:
I waffled on whether to avoid passing `TextAnalyticsClientOptions` directly here, as this is not the `TextAnalyticsClient` type.   For now, I chose to have the actual client prepare scopes and specify defaults rather than having both service clients (legacy/language) duplicate doing so.